graphql_compiler/tests/integration_tests/test_backends_integration.py

```diff
@@ -12,31 +12,60 @@
 from ...tests import test_backend
 from ...tests.test_helpers import generate_schema, generate_schema_graph
 from ..test_helpers import SCHEMA_TEXT, compare_ignoring_whitespace, get_schema
-from .integration_backend_config import MATCH_BACKENDS, SQL_BACKENDS
+from .integration_backend_config import (
+    MATCH_BACKENDS, NEO4J_BACKENDS, REDISGRAPH_BACKENDS, SQL_BACKENDS
+)
 from .integration_test_helpers import (
-    compile_and_run_match_query, compile_and_run_sql_query, sort_db_results
+    compile_and_run_match_query, compile_and_run_neo4j_query, compile_and_run_redisgraph_query,
+    compile_and_run_sql_query, sort_db_results
 )
 
 
-# Store the test parametrization for running against all backends. Individual tests can customize
-# the list of backends to test against with the full @parametrized.expand([...]) decorator.
-all_backends = parameterized.expand([
+all_backends_list = [
     test_backend.ORIENTDB,
     test_backend.POSTGRES,
     test_backend.MARIADB,
     test_backend.MYSQL,
     test_backend.SQLITE,
     test_backend.MSSQL,
-])
+    test_backend.NEO4J,
+    test_backend.REDISGRAPH,
+]
+
+# Store the test parametrization for running against all backends. Individual tests can customize
+# the list of backends to test against with the full @parametrized.expand([...]) decorator.
+all_backends = parameterized.expand(all_backends_list)
```

Comment:
We can also consider this a special case for the `almost_all_backends` decorator with `excluded_backends` as just an empty list. How do we feel about replacing `all_backends` with the `almost_all_backends` decorator? We could rename the `almost_all_backends` decorator as well to make things clearer.

Reply:
I'm hesitant to have two definitions. Also, `almost_all_backends` with no arguments seems like bad design -- it doesn't communicate that it actually doesn't exclude any backends. Why not have one definition, `all_backends` and allow an exclude to be passed to it? Then, if there are no args, it does the obvious thing, and if it's called with a kwarg called e.g. `except_backends` then it also does the obvious thing?

Reply:
https://github.com/kensho-technologies/graphql-compiler/pull/411/files#r304643551